src/ResourceManager/Compute/Commands.Compute/Generated/LogAnalytic/LogAnalyticExportThrottledRequestsMethod.cs

```diff
@@ -0,0 +1,249 @@
+            parameters.FromTime = pFromTime;
+            var pGroupByThrottlePolicy = (bool?) ParseParameter(invokeMethodInputParameters[2]);
+            parameters.GroupByThrottlePolicy = pGroupByThrottlePolicy;
+            var pBlobContainerSasUri = (string) ParseParameter(invokeMethodInputParameters[3]);
+            parameters.BlobContainerSasUri = pBlobContainerSasUri;
+            var pGroupByResourceName = (bool?) ParseParameter(invokeMethodInputParameters[4]);
+            parameters.GroupByResourceName = pGroupByResourceName;
+            var pToTime = (DateTime) ParseParameter(invokeMethodInputParameters[5]);
+            parameters.ToTime = pToTime;
+            string location = (string)ParseParameter(invokeMethodInputParameters[7]);
+
+            var result = LogAnalyticsClient.ExportThrottledRequests(parameters, location);
+            WriteObject(result);
+        }
+    }
+
+    public partial class NewAzureComputeArgumentListCmdlet : ComputeAutomationBaseCmdlet
+    {
+        protected PSArgument[] CreateLogAnalyticExportThrottledRequestsParameters()
+        {
+            ThrottledRequestsInput parameters = new ThrottledRequestsInput();
+            string location = string.Empty;
+
+            return ConvertFromObjectsToArguments(
+                 new string[] { "Parameters", "Location" },
+                 new object[] { parameters, location });
+        }
+    }
+
+    [Cmdlet(VerbsData.Export, "AzureRmLogAnalyticThrottledRequests", DefaultParameterSetName = "DefaultParameter", SupportsShouldProcess = true)]
+    [OutputType(typeof(PSLogAnalyticsOperationResult))]
+    public partial class ExportAzureRmLogAnalyticThrottledRequests : ComputeAutomationBaseCmdlet
+    {
+        public override void ExecuteCmdlet()
+        {
+            ExecuteClientAction(() =>
+            {
+                if (ShouldProcess(this.Location, VerbsData.Export))
+                {
+                    var parameters = new ThrottledRequestsInput();
+                    parameters.GroupByOperationName = this.GroupByOperationName;
+                    parameters.FromTime = this.FromTime;
+                    parameters.GroupByThrottlePolicy = this.GroupByThrottlePolicy;
+                    parameters.BlobContainerSasUri = this.BlobContainerSasUri;
+                    parameters.GroupByResourceName = this.GroupByResourceName;
+                    parameters.ToTime = this.ToTime;
+                    string location = this.Location.Canonicalize();
+
+                    var result = LogAnalyticsClient.ExportThrottledRequests(parameters, location);
+                    var psObject = new PSLogAnalyticsOperationResult();
+                    ComputeAutomationAutoMapperProfile.Mapper.Map<LogAnalyticsOperationResult, PSLogAnalyticsOperationResult>(result, psObject);
+                    WriteObject(psObject);
+                }
+            });
+        }
+
+
```

Comment:
Same comments here about ordering parameters correctly, too many positional parameters, and removing space in SwitchParameters

Reply:
ping